services/src/main/java/org/keycloak/services/managers/RealmManager.java

```diff
@@ -561,6 +561,16 @@ public RealmModel importRealm(RealmRepresentation rep, boolean skipUserDependent
                 realm.setDefaultRole(RepresentationToModel.createRole(realm, rep.getDefaultRole()));
             }
 
+            if (Profile.isFeatureEnabled(Profile.Feature.ADMIN_FINE_GRAINED_AUTHZ_V2)) {
+                if (rep.getAdminPermissionsClient() != null) {
+                    ClientModel client = RepresentationToModel.createClient(session, realm, rep.getAdminPermissionsClient());
```

Comment:
Is it possible to disable this client by default? We should mark this client as either a bearer client (even though I don't like much this type of client, only brings confusion IMO) or a confidential client.

Perhaps we can consider adding an `internal` marker to clients to make sure they cannot request tokens or be used in authentication flows.